code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+			destination_names.Remove(string_to_test)
+
+
+
+/datum/lore/organization/tsc/hephaestus
+	name = "Hephaestus Industries"
+	short_name = "Hephaestus"
+	acronym = "HI"
+	desc = "Hephaestus Industries is the largest supplier of arms, ammunition, and small military vehicles in Sol space. \
+	Hephaestus products have a reputation for reliability, and the corporation itself has a noted tendency to stay removed \
+	from corporate politics. They enforce their neutrality with the help of a fairly large asset-protection contingent which \
+	prevents any contracting polities from using their own materiel against them. SolGov itself is one of Hephastus' largest \
+	bulk contractors owing to the above factors."
+	history = ""
+	work = "arms manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("HTV" = "freight", "HTV" = "munitions resupply")
+	destination_names = list(
+		"a SolGov dockyard on Luna"
+		)
+
+/datum/lore/organization/tsc/vey_med
+	name = "Vey Medical"
+	short_name = "Vey Med"
+	acronym = "VM"
+	desc = "Vey-Med is one of the newer TSCs on the block and is notable for being largely owned and operated by Skrell. \
+	Despite the suspicion and prejudice leveled at them for their alien origin, Vey-Med has obtained market dominance in \
+	the sale of medical equipment-- from surgical tools to large medical devices to the Odysseus trauma response mecha \
+	and everything in between. Their equipment tends to be top-of-the-line, most obviously shown by their incredibly \
+	human-like FBP designs. Vey's rise to stardom came from their introduction of ressurective cloning, although in \
+	recent years they've been forced to diversify as their patents expired and Nanotrasen-made medications became \
+	essential to modern cloning."
+	history = ""
+	work = "medical equipment supplier"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("VTV" = "transportation", "VMV" = "medical resupply")
+	destination_names = list(
+	"a outpost on Jargon 4"
+	)
+
+/datum/lore/organization/tsc/zeng_hu
+	name = "Zeng-Hu pharmaceuticals"
+	short_name = "Zeng-Hu"
+	acronym = "ZH"
+	desc = "Zeng-Hu is an old TSC, based in the Sol system. Until the discovery of Plasma, Zeng-Hu maintained a stranglehold \
+	on the market for medications, and many household names are patentted by Zeng-Hu-- Bicaridyne, Dylovene, Tricordrizine, \
+	and Dexalin all came from a Zeng-Hu medical laboratory. Zeng-Hu's fortunes have been in decline as Nanotrasen's near monopoly \
+	on plasma research cuts into their R&D and Vey-Med's superior medical equipment effectively decimated their own equipment \
+	interests. The three-way rivalry between these companies for dominance in the medical field is well-known and a matter of \
+	constant economic speculation."
+	history = ""
+	work = "pharmaceuticals company"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("ZTV" = "transportation", "ZMV" = "medical resupply")
+	destination_names = list()
+
```

Comment:
/datum/lore/organization/tsc/biotech
	name = "Biotech Solutions"
	short_name = "Biotech"
	acronym = "BTS"
	desc = "A company specializing in the field of synthetic biology, BioTech solutions is at the forefront of providing cutting-edge prosthetics, augmentations, and gene-therapy solutions. Their extensive list of patents and the highly secretive nature of their work often puts them at odds with companies such as Nanotrasen, who commonly reverse-engineer their products. BioTech Solutions is often the victim of industrial sabotage by Cybersun Industries and often relies on planetary governments for asset protection. BioTech Solutions also owns a number of prominent subsidiaries, such as Bishop Cybernetics, Hesphiastos Industries, and Xion Manufacturing Group."
	history = ""
	work = "medical company specializing in prosthetics and pharmaceuticals"
	headquarters = "Xarxis 5"
	motto = ""

	ship_prefixes = list("CIND-T" = "transportation")
	destination_names = list()